src/lwc/rd2ElevateCreditCardForm/rd2ElevateCreditCardForm.js

```diff
@@ -48,6 +52,23 @@ export default class rd2ElevateCreditCardForm extends LightningElement {
         this._paymentMethod = value;
     }
 
+    @api
+    get isEditPayment() {
+        return this._isEditPayment;
+    }
+
+    set isEditPayment(value) {
+        this._isEditPayment = value;
+    }
+
+    get nextPaymentDonationDateMessage() {
+        return (this.nextDonationDate == null)
+        ? ''
+        : '<b>'
+            + this.labels.NextPaymentDonationDateMessage.replace('{{DATE}}', this.nextDonationDate.replace(/(\d{4})\-(\d{2})\-(\d{2})/, '$2/$3/$1'))
```

Comment:
What is the date format being applied here? I believe there is something we can import to handle localization of date formats if necessary.

Reply:
convert from YYYY-MM-DD to MM/DD/YYYY

Reply:
Hmm. That's a US specific format, which might work for today, but the UI should definitely support the users locale. There are various posts on this, but besides `lightning-formatted-date-time` in the template, there's also the `Intl.DateTimeFormat()` [method](https://www.salesforcecodecrack.com/2020/05/how-to-format-dates-in-lightning-web.html) in JS.

Reply:
use lightning-formatted-date-time to take care of local datetime